#This message describes the state of the yoyo
float32 yoyo_pos        #the yoyo z position
float32 yoyo_posvel     #the yoyo z velocity
float32 yoyo_rot        #the yoyo rotation
float32 yoyo_rotvel     #the yoyo rotation velocity